src/runtime/entitlements-manager.js

```diff
@@ -108,12 +124,29 @@ export class EntitlementsManager {
   }
 
   /**
-   * @param {?string=} encryptedDocumentKey
+   * @param {!GetEntitlementsParamsExternal=} params
    * @return {!Promise<!Entitlements>}
    */
-  getEntitlements(encryptedDocumentKey) {
+  getEntitlements(params) {
```

Comment:
The backend expects a encryptedDocumentKey and a param at the moment, so it would be good to put it separately for now otherwise I think it will break encryption items.

Reply:
Don't worry, the JavaScript is still sending the encryption parameter to the backend just as it used to. You can search this file for "crypt=" to verify.

Reply:
Awesome work! Disregard my initial comment.